src/main/java/io/lighty/yang/validator/formats/JsTree.java

```diff
@@ -55,145 +55,182 @@
-                    RpcInputOutput inputOutputOther = RpcInputOutput.OTHER;
-                    for (int i = 1; i <= qnames.size(); i++) {
-                        List<QName> qnamesCopy = new ArrayList<>(qnames);
-                        qnamesCopy = qnamesCopy.subList(0, i);
-                        if (!actions.isEmpty()) {
-                            for (ActionDefinition action : actions) {
-                                if (action.getQName().getLocalName()
-                                        .equals(qnamesCopy.get(qnamesCopy.size() - 1).getLocalName())) {
-                                    if (INPUT.equals(qnames.get(i).getLocalName())) {
-                                        inputOutputOther = RpcInputOutput.INPUT;
-                                    } else {
-                                        inputOutputOther = RpcInputOutput.OUTPUT;
-                                    }
-                                }
-                            }
-                        }
-                        final ListIterator<Integer> integerListIterator =
-                                removeChoiceQnames.listIterator(removeChoiceQnames.size());
-                        while (integerListIterator.hasPrevious()) {
-                            qnamesCopy.remove(integerListIterator.previous().intValue());
-                        }
-                        if (!this.schemaContext.findDataTreeChild(qnamesCopy).isPresent()) {
-                            removeChoiceQnames.add(i - 1);
-                        } else if (this.schemaContext.findDataTreeChild(qnamesCopy).get()
-                                instanceof ActionNodeContainer) {
-                            final ActionNodeContainer actionSchemaNode =
-                                    (ActionNodeContainer) this.schemaContext.findDataTreeChild(qnamesCopy).get();
-                            actions = actionSchemaNode.getActions();
-                        }
-                    }
-                    ids.add(modelAugmentationNumber++);
-                    HtmlLine line = new HtmlLine(new ArrayList<>(ids), node, inputOutputOther, this.schemaContext,
-                            removeChoiceQnames, namespacePrefix, Optional.empty(), false);
-                    lines.add(line);
-                    resolveChildNodes(lines, new ArrayList<>(ids), node, RpcInputOutput.OTHER, removeChoiceQnames,
-                            Collections.emptyList());
-                    ids.remove(ids.size() - 1);
-                }
-                for (Line line : lines) {
-                    final String linesText = line.toString();
-                    LOG.info(linesText);
-                }
-                lines = new ArrayList<>();
+                printLines(getAugmentationNodesLines(singletonListInitializer.getSingletonListWithIncreasedValue(),
+                                                     augNode, removeChoiceQnames));
```

Comment:
please use continuation indent: `8`, do not align to first parameter

Reply:
done